docs/tutorial.rst

```diff
@@ -27,17 +27,18 @@ Or you can be extra fancy and use `pipx`_:
 
 Either way, Nox is usually installed *globally*, similar to ``tox``, ``pip``, and other similar tools.
 
-If you're interested in running ``nox`` within `docker`_, you can use the `thekevjames/nox images`_ on DockerHub which contain builds for all ``nox`` versions and all supported ``python`` versions.
+If you're interested in running ``nox`` within `docker`_, you can use the `thekevjames/nox images`_ on DockerHub which contain builds for all ``nox`` versions and all supported ``python`` versions. Nox is also supported via ``pipx run nox`` in the `manylinux images`_.
 
-If you want to run ``nox`` within `GitHub Actions`_, you can use the `excitedleigh/setup-nox action`_, which installs the latest ``nox`` and makes available all Python versions provided by the GitHub Actions environment.
+If you want to run ``nox`` within `GitHub Actions`_, you can use the `wntrblm/nox action`_, which installs the latest ``nox`` and makes available all active CPython and PyPY versions provided by the GitHub Actions environment. You can safely combine this with with ``setup-python`` for past end-of-life or development versions of Python, as well.
 
 .. _pip: https://pip.readthedocs.org
 .. _user site: https://packaging.python.org/tutorials/installing-packages/#installing-to-the-user-site
 .. _pipx: https://packaging.python.org/guides/installing-stand-alone-command-line-tools/
 .. _docker: https://www.docker.com/
 .. _thekevjames/nox images: https://hub.docker.com/r/thekevjames/nox
 .. _GitHub Actions: https://github.com/features/actions
-.. _excitedleigh/setup-nox action: https://github.com/marketplace/actions/setup-nox
+.. _wntrblm/nox action: https://github.com/marketplace/actions/nox
```

Comment:
FYI, this link assumes you click "publish this action" after this in GitHub Actions. In cibuildwheel, we've not done that (to avoid a "this is an action in the marketplace" banner at the top to make it look like it's only GitHub Actions). If you'd prefer not to publish to the marketplace" (which doesn't really do anything other than make it a tiny bit more searchable), then this link should point somewhere else, or not be a link at all.

I did check to make sure the name "nox" has not been taken yet.

Reply:
I can't see any harm in publishing it in this way, it'd be nice to have an official nox action in the marketplace :) If any other maintainers feel differently though please chime in!

Reply:
It just clicked what you meant by this @henryiii! For anyone else as slow to catch on as me, this is what we mean: 

<img width="938" alt="image" src="https://user-images.githubusercontent.com/62767721/165464733-ef93fdd9-8898-4323-9dba-c03fa9222f76.png">

If we publish the action in the marketplace, the Nox repo will have that banner above it which implies it's just a GitHub action so IMO it's best to avoid as @henryiii did with cibuildwheel